api/firmware/eth.go

```diff
@@ -70,6 +70,7 @@ func (device *Device) ETHPub(
 }
 
 // ETHSign signs an ethereum transaction. It returns a 65 byte signature (R, S, and 1 byte recID).
+// 27 is added to the recID to denote an uncompressed pubkey.
```

Comment:
I'm confused. I thought this was about signing a message and so expected the changes to be in `ETHSignMessage`.

Reply:
Ooh that was a bad blunder!! Will fix.

Reply:
Done